### PR TITLE
Update NotificationView toast fonts

### PR DESCRIPTION
**Pull request by laminesm**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

Notification was updated to use body 2 font for subtext and single text toast.
Footnote font is now unused so it was removed as well.

### Verification

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| before-light | after-light |
| before-dark | after-dark |
| before_light | Screen Shot 2022-08-19 at 2 46 25 PM |
| before_dark | Screen Shot 2022-08-19 at 2 46 31 PM |
| Screen Shot 2022-08-19 at 2 35 19 PM | after_swiftui_light |
| before_swiftui_dark | after_swiftui_dark |
| before_primary_light | after_primary_light |
| before_primary_dark | after_primary_dark |
| before_neutral_light | after_neutral_light |
| before_neutral_dark | after_neutral_dark |
| before_danger_light | after_danger_light |
| before_danger_dark | after_danger_dark |
| before_warning_light | after_warning_light |
| before_warning_dark | after_warning_dark |
| Screen Shot 2022-08-19 at 7 49 37 PM | after_light |
| Screen Shot 2022-08-19 at 7 49 43 PM | after_dark |

### Pull request checklist

This PR has considered:
- [x] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1176)

### CODE REVIEW

**Comment by harrieshin** [comment]
screenshot of a toast that has both title and message?

File: ios/FluentUI/Notification/FluentNotification.swift [hunk 1]
```diff
@@ -164,7 +164,7 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
                     }
                     .accessibilityLabel(attributedMessage.string)
             } else if let message = state.message {
-                let messageFont = hasSecondTextRow ? tokens.footnoteTextFont : (state.style.isToast ? tokens.boldTextFont : tokens.regularTextFont)
+                let messageFont = tokens.regularTextFont
```
Comment (harrieshin) [inline ios/FluentUI/Notification/FluentNotification.swift:167]: do we need this local variable? just directly set it like titlelabel?

**Comment by laminesm** [comment]
> screenshot of a toast that has both title and message?

Description is updated with screenshots.

**Review by mischreiber — APPROVED** [review]

**Review by harrieshin — APPROVED** [review]

File: ios/FluentUI/Notification/FluentNotification.swift [hunk 2]
```diff
@@ -164,9 +164,8 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
                     }
                     .accessibilityLabel(attributedMessage.string)
             } else if let message = state.message {
-                let messageFont = hasSecondTextRow ? tokens.footnoteTextFont : (state.style.isToast ? tokens.boldTextFont : tokens.regularTextFont)
                 Text(message)
-                    .font(.fluent(messageFont))
+                    .font(.fluent(tokens.regularTextFont))
```
Comment (huwilkes) [inline ios/FluentUI/Notification/FluentNotification.swift:168]: Do we need to add a check to Title to make sure we aren't making it bold when there's only one line of text?

**Comment by huwilkes** [comment]
Could you add a screenshot of at least one bar style?

File: ios/FluentUI/Notification/FluentNotification.swift (hunk 2 above)
Comment (laminesm) [reply ios/FluentUI/Notification/FluentNotification.swift:168]: This change takes care of that as it always sets the subtext to body 2.

**Pull request merged**